.github/workflows/scripts/env.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# This file installs common linux environment tools
+
+export LANG C.UTF-8
+
+# python_version=$1
+
+sudo    apt-get update && \
+sudo    apt-get install -y --no-install-recommends \
+        software-properties-common \
+
+sudo    apt-get install -y --no-install-recommends \
+        build-essential \
+        apt-utils \
+        ca-certificates \
+        wget \
+        git \
+        vim \
+        libssl-dev \
+        curl \
+        unzip \
+        unrar \
+        cmake \
+        net-tools \
+        sudo \
+        autotools-dev \
+        rsync \
+        jq \
+        openssh-server \
+        tmux \
+        screen \
+        htop \
+        pdsh \
+        openssh-client \
+        lshw \
+        dmidecode \
+        util-linux \
+        automake \
+        autoconf \
+        libtool \
+        net-tools \
+        pciutils \
+        libpci-dev \
+        libaio-dev \
+        libcap2 \
+        libtinfo5 \
+        fakeroot \
+        devscripts \
+        debhelper \
+        nfs-common
+
+# Install python and pip
+# sudo add-apt-repository ppa:deadsnakes/ppa
+# sudo apt install "python$python_version" "python$python_version-distutils" "python$python_version-dev"
+# wget https://bootstrap.pypa.io/get-pip.py
+# python$python_version get-pip.py
+# rm get-pip.py
```

Comment:
Did you leave these lines on any purpose? If not, could you delete these?

Reply:
Yes we can delete them. The python installation step didn't work properly at first. This was the first fix, but later I managed to get it to work :P

Reply:
At some point I encountered a problem where some wheels failed to upload due to an unknown reason. After some debugging I found out there was a name collision between the wheels. At first I thought there was a problem with the action so I tried to replace it. Upon further investigation I discovered there was a colission between the python that was pre-installed with the new one, and by default it took the pre-installed one. This is the reason some of the scripts define `python_executable=python$1`

Reply:
Got it. Thanks for the detailed explanation!